cargo-dist/src/config.rs

```diff
@@ -980,9 +980,9 @@ impl<'de> Deserialize<'de> for ZipStyle {
             ".zip" => Ok(ZipStyle::Zip),
             ".tar.gz" => Ok(ZipStyle::Tar(CompressionImpl::Gzip)),
             ".tar.xz" => Ok(ZipStyle::Tar(CompressionImpl::Xzip)),
-            ".tar.zstd" => Ok(ZipStyle::Tar(CompressionImpl::Zstd)),
```

Comment:
for backwards compatibility, we'll probably want to keep this around for at least a few releases. would you mind updating to accept both `zstd` and `zst`?